packages/adapter-vercel/index.d.ts

```diff
@@ -1,7 +1,40 @@
 import { Adapter } from '@sveltejs/kit';
 import './ambient.js';
 
-export default function plugin(config?: Config): Adapter;
+export default function plugin(
+	config?: Config & {
+		/**
+		 * Enable or disable Vercel's image optimization. This is enabled by default if you have
+		 * defined the Vercel loader in `kit.images.loader`, else disabled by default.
+		 * https://vercel.com/docs/concepts/image-optimization
+		 */
+		images?: ImageConfig;
+	}
+): Adapter;
+
+/**
+ * https://vercel.com/docs/concepts/image-optimization
```

Comment:
I think this might be a better link to use. or we could include both
```suggestion
 * https://vercel.com/docs/build-output-api/v3/configuration#images
```